### DISCUSSION TITLE
Emulator crash

### DISCUSSION THREAD

**Discussion by deepankar2121** [skyline-emu/skyline#831] [General]
Skyline emulator crash on My device Samsung M30s 
OS-   Android 11, One UI 
 3.0ChipsetExynos 9611 (10nm)
CPU-Octa-core (4x2.3 GHz Cortex-A73 & 4x1.7 GHz Cortex-A53)GPU
Mali-G72 MP3
Please fix the crash of emulator

**Comment by PixelyIon** [comment]
This is likely the same as #416. Just track that issue.
